lib/docblockr-worker.js

```diff
@@ -296,6 +311,27 @@ module.exports =
       this.write(editor, snippet);
     };
 
+    DocBlockrAtom.prototype.parse_python_command = function(inline) {
```

Comment:
All the python specific functions should better be in a language specific file.

Reply:
Okay, i can add a parse_command in the language parser if you want. Do you think that it will be maintainable ? Thus i would add the current parse command function in the parent language parser class